### PR TITLE
Getting Started Guide landing page

### PR DESCRIPTION
**Pull request by hobbes-the-tiger**
Created a new landing page for the documentation site.

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -2,60 +2,48 @@
...
+[direct ingest using our built-in HTTP API]({{< ref "developer-resources/integrations/data-shippers/hec.md" >}}), 
+so it is easy to adopt or migrate to and from other platforms. 
 
-## Integrations
+### Step 3: Configuring Parsers
 
-<a href="{{% ref "integrations/_index.md" %}}">Full List of Integrations</a>
+Once you have log data flowing into Humio, you need to parse it to make the data in those logs more usable in 
+queries. [Parsing]({{< ref "operations-guide/parsers/" >}}) lets you take logs that are essentially lines of text, 
+and extract their structure in order to incorporate a more in-depth analysis.
```
Comment (jpease) [inline content/_index.md]: Is "incorporate" the right word? What are we adding a more in-depth analysis to?

File: content/_index.md [hunk 2]
```diff
@@ -2,60 +2,48 @@
...
+[Metricbeat]({{< ref "developer-resources/integrations/data-shippers/beats/metricbeat.md" >}}), and
+[direct ingest using our built-in HTTP API]({{< ref "developer-resources/integrations/data-shippers/hec.md" >}}), 
+so it is easy to adopt or migrate to and from other platforms. 
 
-## Integrations
+### Step 3: Configuring Parsers
 
-<a href="{{% ref "integrations/_index.md" %}}">Full List of Integrations</a>
+Once you have log data flowing into Humio, you need to parse it to make the data in those logs more usable in 
+queries. [Parsing]({{< ref "operations-guide/parsers/" >}}) lets you take logs that are essentially lines of text,
```
Comment (jpease) [inline content/_index.md]: Stylistic, so feel free to ignore, but "essentially" and "just" are red flags for me.

Would this be simpler as: "Parsing lets you take plain-text logs and extract their structure to..."

File: content/_index.md [hunk 3]
```diff
@@ -2,60 +2,48 @@
...
 
-The REST API and snappy graphical user interface allows you to search, transform and observe
-events as they occur – in Real-Time. And when we say real-time, we mean sub-second delays
-from data arriving for ingestion before it is displayed on a live dashboard.
+### Step 1: Setting up Humio on the cloud or on bare metal
 
-Finally, built-in support for alerting and integrations with popular operations platforms (like
-PagerDuty, Opsgenie or VictorOps) means Humio can be the backbone of your operations infrastructure,
-a power tool for your help-desk teams and your trusted navigator on an ocean of data.
+Humio is easy to deploy. You can [deploy Humio on your own servers](https://www.humio.com/download/) or
```
Comment (jpease) [inline content/_index.md]: The first and third sentences seem somewhat duplicative. Would this be an improvement?

Humio is easy to deploy. You can deploy Humio on your own servers or cloud infrastructure, or you can let Humio take care of the hosting and management in our cloud.

Note: Links omitted.

File: content/_index.md (hunk 1 above)
Comment (hobbes-the-tiger) [reply content/_index.md]: @jpease I would think that's the right word. We are incorporating more in-depth analysis to their data (logs, etc).

LH-ExcellentAdventure What's your thoughts on this one?

File: content/_index.md (hunk 2 above)
Comment (hobbes-the-tiger) [reply content/_index.md]: Agreed. Fixed.

File: content/_index.md (hunk 3 above)
Comment (hobbes-the-tiger) [reply content/_index.md]: Already fixed from the Google doc. Thanks! :)

File: content/_index.md (hunk 1 above)
Comment (LH-ExcellentAdventure) [reply content/_index.md]: I can argue for incorporate, but I think "achieve" might be a better word in that spot. @jpease, does that work for you?

Comment (hobbes-the-tiger) [reply content/_index.md]: I like "achieve" better as well. Do you want me to change it @LH-ExcellentAdventure?

**Pull request merged**